tests/zfs-tests/tests/functional/cli_root/zpool_import/zpool_import_errata3.ksh

```diff
@@ -62,7 +62,8 @@ log_assert "Verify that Errata 3 is properly handled"
 
 uncompress_pool
 log_must zpool import -d /$TESTPOOL/ $POOL_NAME
-log_must eval "zpool status | grep -q Errata"
+log_must eval "zpool status $POOL_NAME | grep -q Errata"
+log_must eval "zpool status $POOL_NAME | grep -q ZFS-8000-ER"
```

Comment:
nit: perhaps other msgid table values should be tested for (where the pool status is already being produced as part of a test).

Reply:
That sounds like a huge overhaul, could we maybe do that in another PR?  @loli10K any thoughts?

Reply:
I think it's reasonable to do it separately.  It's not necessarily easy to find the tests that already produce these states, even if they already exist.